acls.go

```diff
@@ -444,7 +444,7 @@ func expandGroup(
 				errInvalidGroup,
 			)
 		}
-		grp, err := NormalizeNamespaceName(group, stripEmailDomain)
+		grp, err := NormalizeName(group, stripEmailDomain)
```

Comment:
I think as mentioned this is a bit to generic, for a unknown user to understand the context, it needs to either have `FQDN` or Domain or some reference to the RFC and limitations in the name.